tsdb/index/postings.go

```diff
@@ -397,6 +397,40 @@ func (p *MemPostings) addFor(id storage.SeriesRef, l labels.Label) {
 	}
 }
 
+func (p *MemPostings) PostingsForMatcher(ctx context.Context, pr PostingsReader, m *labels.Matcher) Postings {
+	if p, ok := fastPostingsForMatcher(ctx, pr, m); ok {
+		return p
+	}
+
+	p.mtx.RLock()
+
+	e := p.m[m.Name]
+	if len(e) == 0 {
+		p.mtx.RUnlock()
+		return EmptyPostings()
+	}
+
+	// Benchmarking shows that first copying the values into a slice and then matching over that is
+	// faster than matching over the map keys directly, at least on AMD64.
+	// However, when using a Swiss Table implementation instead of a standard map, it's performant.
```

Comment:
This is the only mention of Swiss Table I can see in this PR.  Why are we mentioning it here?

Reply:
I can remove this line about Swiss Table. I just wanted to record that I couldn't see this performance issue with Swiss Table, in case someone Prometheus would use Swiss Table for maps in the future.

Should I remove the Swiss Table line?

Reply:
If it has nothing to do with this PR I would prefer to have it removed.

Reply:
Will do.

Reply:
Done.